src/runtime/storage-proxy.ts

```diff
@@ -556,7 +557,7 @@ export class BigCollectionProxy extends StorageProxy implements BigCollectionSto
 export class StorageProxyScheduler {
   private _scheduled = false;
   private _queues = new Map<Particle, Map<Handle, [string, Particle, {}][]>>();
-  private _idleResolver: (() => void) | null = null;
+  private _idleResolver: Runnable | null = null;
```

Comment:
If we add the `Optional` type above, it could be used here, too.

Reply:
Agreed, sounds like a good follow on PR, since this value flips between being set and being null.